components/compositing/compositor.rs

```diff
@@ -615,20 +615,18 @@ impl<Window: WindowMethods> IOCompositor<Window> {
             }
 
             (Msg::CollectMemoryReports(reports_chan), ShutdownState::NotShuttingDown) => {
-                let mut reports = vec![];
                 let name = "compositor-task";
                 // These are both `ExplicitUnknownLocationSize` because the memory might be in the
                 // GPU or on the heap.
-                reports.push(mem::Report {
+                let mut reports = vec![mem::Report {
```

Comment:
@rnestler Yes, that is what I had intended to write. I've just removed `mut` from the branch.